test_launch_ros/test/test_launch_ros/frontend/test_node_frontend.py

```diff
@@ -26,12 +27,17 @@
 import pytest
 
 yaml_params = str(pathlib.Path(__file__).parent / 'params.yaml')
+
+# Escape backslashes if any to keep them after parsing takes place
+yaml_params = yaml_params.replace('\\', '\\\\')
```

Comment:
Could you use `as_posix` instead of `str()` above to avoid this?

https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.as_posix

Reply:
Hmm, I don't have a Windows box at hand to try it out, but if we do that then that POSIX path will be forwarded to the underlying node. And `rcl` will use `fopen` to attempt to parse the file with that path. And I think `fopen` will not handle POSIX paths in Windows.

Reply:
I see.

Reply:
> Hmm, I don't have a Windows box at hand to try it out, but if we do that then that POSIX path will be forwarded to the underlying node. And `rcl` will use `fopen` to attempt to parse the file with that path. And I think `fopen` will not handle POSIX paths in Windows.

I'm almost sure that most native Python functions that takes paths accept both posix and windows' style paths.
I remember I tested that recently and it worked.
Particularly, I think I tested `open`, but it's better to double check.